bot/exts/internal_eval/_internal_eval.py

```diff
@@ -0,0 +1,182 @@
+import logging
+import re
+import textwrap
+import typing
+
+import discord
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Roles
+from bot.utils.decorators import with_role
+from ._helpers import EvalContext
+
+__all__ = ["InternalEval"]
+
+log = logging.getLogger("rattlesnake.exts.admin_tools.internal_eval")
```

Comment:
The logger name needs to be updated. It should even be dynamic.
```suggestion
log = logging.getLogger(__name__)
```